arangod/Aql/EngineInfoContainerDBServerServerBased.cpp

```diff
@@ -82,7 +82,6 @@ EngineInfoContainerDBServerServerBased::TraverserEngineShardLists::TraverserEngi
   auto const& restrictToShards = query.queryOptions().restrictToShards;
   // Extract the local shards for edge collections.
   for (auto const& col : edges) {
-    TRI_ASSERT(col != nullptr);
```

Comment:
I think the changes in this file should all be reverted. They have probably slipped into this PR by mistake.

Reply:
Good catch - done!